src/lexer/lexer.cpp

```diff
@@ -23,21 +23,42 @@ int Lexer::GetToken(std::ifstream &is) {
       identifier_string_ += last_char;
     }
     if (identifier_string_ == "int") {
+      current_datatype_ = "int";
```

Comment:
current_datatype_  = identifier_string_
for create const static strings.
Just a suggestion.

Reply:
What is to be defined static ?

Reply:
resolve